GameServerLib/Maps/SummonersRift.cs

```diff
@@ -803,10 +803,10 @@ public bool Spawn()
                 }
 
                 List<Vector2> waypoint = new List<Vector2>();
-                TeamId team = barrack.Value.GetTeamID();
+                TeamId team = barrack.Value.GetOpposingTeamID();
```

Comment:
Change this variable to `oppose_team` as it can be confusing.

Reply:
Resolved